source/ns_cmdline.c

```diff
@@ -1892,3 +2101,51 @@ char *cmd_parameter_last(int argc, char *argv[])
     }
     return NULL;
 }
+/**
+ * find last space but ignore nulls and first spaces.
+ * used only internally to find out previous word
+ * e.g.
+ * const char str[] = "aaaab aa bbb\0\0";
+ * const char* tmp = last_space(str+strlen(str), str);
+ * printf(tmp); // prints "bbb"
+ */
+static const char* find_last_space(const char* from, const char* to)
+{
+  if (from <= to) return 0;
+  while((from > to) && ((*from == 0) || (*from == ' '))) {
+    from--;
+  }
+  while (from > to) {
+    if (*from == ' ') {
+      return from+1;
```

Comment:
So actually this returns a pointer to the character _following_ the last space (ignoring trailing spaces).

Alright, makes sense now.

Reply:
yep